test/parallel/test-cluster-debug-port-manual.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/**
+ * This test suite checks debug port numbers
+ * when node started without --debug argument
+ * and debug initiated with cluster.settings.execArgv or similar
+ */
+
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+if (cluster.isMaster) {
+  function checkExitCode(code, signal) {
+    assert.strictEqual(code, 0);
+    assert.strictEqual(signal, null);
+  }
+
+  // If master is not debugging,
+  // forked worker should not have --debug argument
+  cluster.fork().on('exit', common.mustCall(checkExitCode));
+
+  cluster.settings.execArgv = ['--debug=13000'];
```

Comment:
Please use `common.PORT`, if necessary with an offset.  Fixed port numbers won't work when the CI does concurrent test suite runs.